flink-java/src/main/java/org/apache/flink/api/java/typeutils/PojoTypeInfo.java

```diff
@@ -310,6 +316,18 @@ public int getFieldIndex(String fieldName) {
 
 	@Override
 	public TypeSerializer<T> createSerializer(ExecutionConfig config) {
+		if(config.isForceKryoEnabled() && config.isForceAvroEnabled()) {
```

Comment:
Okay. I'll move it to the other executionconfig related logging at the executionenvironment.